src/main/java/net/sourceforge/plantuml/activitydiagram3/ftile/UGraphicInterceptorUDrawable2.java

```diff
@@ -106,15 +110,17 @@ private void drawGoto(FtileGoto ftile) {
 			return;
 		final double dx = dest.getDx() - posNow.getDx();
 		final double dy = dest.getDy() - posNow.getDy();
+		if (isDebug) {
 		ugGoto.draw(UEllipse.build(3, 3));
 		ugGoto.apply(new UTranslate(dx, dy)).draw(UEllipse.build(3, 3));
+		}
 		ugGoto.draw(ULine.hline(dx));
 		ugGoto.apply(UTranslate.dx(dx)).draw(ULine.vline(dy));
 		// ugGoto.draw(new ULine(dx, dy));
 	}
 
 	public UGraphic apply(UChange change) {
-		return new UGraphicInterceptorUDrawable2(getUg().apply(change), positions);
+		return new UGraphicInterceptorUDrawable2(getUg().apply(change), positions, gotoColor, isDebug);
```

Comment:
- Do you should propagate also `gotoColor` to `drawGoto`?

Reply:
> Do we should propagate also `gotoColor` to `drawGoto`?

You don't have to, since gotoColor is stored as field in the class:

https://github.com/plantuml/plantuml/blob/6f32e332c5bf71d5bb4eb7d19313926c21a364c2/src/main/java/net/sourceforge/plantuml/activitydiagram3/ftile/UGraphicInterceptorUDrawable2.java#L57

But maybe I don't understand your question...